### PR TITLE
[doc]add FAQ

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
### What changes were proposed in this pull request?
1. Add FAQ
2. Update TOC doc.
3. Merge Trouble Shooting into FAQ.

### CODE REVIEW

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+| 100         |
+---------------
+| 102         |
+---------------
+| 125         |
+---------------
+Got 3 rows (Time spent: 7431/10406 us)
+```
+
+Taking the above query as an example, the first number is the time it takes the query engine to receive the data from the console and pass the data to the storage engine to perform a series of calculations on the entire link; the second number is the time spent from sending the request to receiving the response and outputting the results in the console.
```
Comment (whitewum) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md]: `Time spent: 7431/10406 us`, the first number (`7431`) is the time consuming of the database.  i.e., the sum of a) receiving the query from CLI, b)  asking and getting back results from the storage engine (storaged),  c)  finishing computation in graphd; 
the second number (10406) is the time consuming from CLI's view. i.e., the sum of the above time plus d) request back to the CLI, and e) displaying the results on the screen.

File: docs/manual-CN/README.md [hunk 2]
```diff
@@ -22,6 +22,7 @@
   * [导入 .csv 文件](3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/import-csv-file.md)
   * [加载 .sst 文件](3.build-develop-and-administration/3.deploy-and-administrations/server-administration/storage-service-administration/data-import/download-and-ingest-sst-file.md)
   * [Nebula Graph 客户端](1.overview/2.quick-start/3.supported-clients.md)
+  * [常见问题](1.overview/2.quick-start/2.FAQ.md)
```
Comment (whitewum) [inline docs/manual-CN/README.md]: 常见问题FAQ

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+| 100         |
+---------------
+| 102         |
+---------------
+| 125         |
+---------------
+Got 3 rows (Time spent: 7431/10406 us)
+```
+
+以上述查询为例，Time spent 中前一个数字为 query engine 从收到 console 这条语句到存储拿到数据并进行一系列计算整条链路花的时间；后一个数字是 console 从发送请求到收到响应并将结果输出到控制台的时间。
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: 以上述查询为例，Time spent 中前一个数字 `7431` 为 数据库本身 所花费的时间，即 query engine 从收到 console 这条查询语句，到存储拿到数据, 并进行一系列计算所花的时间；后一个数字 `10406` 是从客户端角度看花费的时间，即console 从发送请求，到收到响应，并将结果输出到屏幕的时间。

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+```ngql
+nebula> UPDATE CONFIGS storage:load_data_interval_secs=1
+nebula> UPDATE CONFIGS graph:load_data_interval_secs=1
+```
+
+> Note the changes take effect in the next period.
+
+### Errors Thrown When Executing Command in Docker
+
+This is likely caused by the inconsistency between the docker IP and the default listening address (172.17.0.2). Thus we need to change the the latter one.
```
Comment (wilsonyou) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md]: latter one -> latter

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+1. First run `ifconfig` in container to check your container IP, here we assume your IP is 172.17.0.3.
+2. In directory `/usr/local/nebula/etc`, check the config locations of all the IP addresses with the command `grep "172.17.0.2" . -r`.
+3. Change all the IPs you find in step 2 to your container IP 172.17.0.3.
+4. Restart all the services.
+
+### How to Check Logs
+
+Logs are stored under `/usr/local/nebula/logs/` by default.
+
+Logs details please refer to [logs](docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md).
```
Comment (wilsonyou) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md]: For details of logs, please refer to.....

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+### How to Check Runtime Configs
+
+In Nebula console, run
+
+```ngql
+nebula> SHOW CONFIGS;
+```
+
+Configuration details please see [here](docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md).
```
Comment (wilsonyou) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md]: For configuration details, please see


Please change similar issues

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+| 100         |
+---------------
+| 102         |
+---------------
+| 125         |
+---------------
+Got 3 rows (Time spent: 7431/10406 us)
+```
+
+以上述查询为例，Time spent 中前一个数字 `7431` 为数据库本身所花费的时间，即 query engine 从收到 console 这条查询语句，到存储拿到数据，并进行一系列计算所花的时间；后一个数字 `10406` 是从客户端角度看花费的时间，即 console 从发送请求，到收到响应，并将结果输出到屏幕的时间。
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: query engine 从 console 收到这条查询语句，到存储拿到数据，并进行一系列计算所花的时间；

Comment (Amber1990Zhang) [reply docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: nice catch

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+| 100         |
+---------------
+| 102         |
+---------------
+| 125         |
+---------------
+Got 3 rows (Time spent: 7431/10406 us)
+```
+
+Take the return results (`Time spent: 7431/10406 us`) of the above query as example, the first number (`7431`) is the time consuming of the database. i.e., the sum of a) receiving the query from CLI, b) asking and getting back results from the storage engine (storaged), c) finishing computation in graphd; the second number (`10406`) is the time consuming from the CLI's view. i.e., the sum of the above time plus d) request back to the CLI, and e) displaying the results on the screen.
```
Comment (wilsonyou) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md]: Taking the above query as an example, the number `7431` in Time spent is the time spent by the database itself, that is, the time it takes for the query engine to receive a query statement from the console, fetch the data from the storage and perform a series of calculation ; the number `10406` is the time spent from the client's perspective, that is, the time it takes for the console from sending a request and receiving a response to displaying the result on the screen.

File: docs/manual-CN/1.overview/2.quick-start/2.FAQ.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+可能原因， `load_data_interval_secs` 设置了从 meta server 获取元数据时间间隔。更改方式:
+
+如果 meta 注册过配置，请在 console 中运行以下命令查看参数 `load_data_interval_secs` 的值。
+
+```ngql
+nebula> GET CONFIGS storage:load_data_interval_secs
+nebula> GET CONFIGS graph:load_data_interval_secs
+```
+
+如果值过大，运行以下命令更改将值更改为 1s
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.FAQ.md]: 运行以下命令将值更改为 1s

**Pull request merged**